packages/core/src/components/Chip/Chip.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import * as React from 'react';
+import cx from "classnames";
+import css from "./Chip.css";
+
+type
+Props = {
+  className?: string,
+  href: string,
+  text: string,
+};
+
+const Chip = ({className, href, text}: Props) => (
+  <a href={href} className={cx(css.root, className)}>
+    <div>
+      {text}
+    </div>
```

Comment:
this div seems unnecessary:
```suggestion
    {text}
```